utils/build/docker/nodejs/express4/app.js

```diff
@@ -250,7 +251,37 @@ iast.initRoutes(app, tracer)
 require('./auth')(app, passport, tracer)
 require('./graphql')(app)
 
+// try to flush as much stuff as possible from the library
+app.get('/flush', (req, res) => {
+  // doesn't have a callback :(
+  // tracer._tracer?._dataStreamsProcessor?.writer?.flush?.()
```

Comment:
TODO: for some reason running this function makes the schema non regression tests fail ???